README.md

```diff
@@ -89,13 +90,18 @@ violations for your environment.
 --    url - The base URL to iprepd (defaults to "http://localhost:8080/")
 --    cache_ttl - The iprepd response cache ttl in seconds (defaults to 30)
 --    timeout - The timeout for making requests to iprepd in milliseconds (defaults to 10)
+--    cache_errors - Enables or disables caching errors. Caching errors will make is so
+--                   that the average additional latency added by this module in the 0.5ms
+--                   range (as long as there is a reasonable cache ttl), but can make
+--                   testing harder.
```

Comment:
If it makes testing harder should it be enabled by default?

Reply:
heh yeah, agreed.